### DISCUSSION TITLE
Many to many participants

### DISCUSSION THREAD

**Discussion by younos1986** [pion/webrtc#1347] [General] [answered]
Is there a feature of many to many participants in PION? I mean a many to many call based on Star topology.
If so, is there any sample or just share the big idea!

thanks

**Comment by OrlandoCo** [comment, marked as answer]
check pion/ion from this repository is a SFU media server, also in ion/examples there is a basic SFU server

**Reply by Sean-Der** [reply]
@younos1986 For most users I think Ion, peer-calls, kraken.fm and Galene are your best bet.

If you wanted to really build things from scratch [sfu-ws](https://github.com/pion/example-webrtc-applications/tree/master/sfu-ws) is a good starting point.
